beta/project-griffon/using-project-griffon.md

```diff
@@ -13,55 +13,51 @@ Project Griffon is in beta. Use of this beta product requires acceptance of term
 If you do not know your Adobe ID credentials, contact your Adobe administrator or see [how to log in](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/getting-started-experience-cloud.html).
 {% endhint %}
 
-![](../../.gitbook/assets/image-2.png)
+3. In the beta access page, click **Register**. 
 
-If you see the following screen, for beta access, click **Register**.
+## Project Griffon sessions
 
-![](../../.gitbook/assets/image-7.png)
+To begin a session, you need to create a session and connect your device.
 
-## Begin a Project Griffon session
+### 1.Create a session
 
-### Create a session
+1. Log in to Project Griffon. 
```

Comment:
If it's not too much of a pain, can we change the step numbers to letters? (Instead of "1.", change it to "a."